src/OSPSuite.Infrastructure/Configuration/OSPSuiteConfiguration.cs

```diff
@@ -42,19 +42,21 @@ public abstract class OSPSuiteConfiguration : IApplicationConfiguration
       public abstract string WatermarkOptionLocation { get; }
       public abstract string ApplicationFolderPathName { get; }
       public virtual string LicenseAgreementFilePath { get; } = Constants.Files.LICENSE_AGREEMENT_FILE_NAME;
+      private readonly bool _isReleasedVersion;
 
       protected OSPSuiteConfiguration()
       {
          var assemblyVersion = AssemblyVersion;
          MajorVersion = version(assemblyVersion.Minor);
          Version = $"{MajorVersion}.{assemblyVersion.Build}";
          ReleaseDescription = retrieveReleaseDescription();
```

Comment:
Released Description used to be empty in release and EAP , alpha beta otherwise.
Now this will changed. It will be something like 7.2.3.4 in release and 7.2.3.4 EAP in EAP

The reason is that many tools read the value of AssemblyInformatiomal Attribute and if it's there, use it instead of the Version. That's why the version shown in the CLI is EAP or empty